sdk/common/src/main/scala/org/typelevel/otel4s/sdk/autoconfigure/TelemetryResourceAutoConfigure.scala

```diff
@@ -80,9 +80,12 @@ private final class TelemetryResourceAutoConfigure[F[_]: MonadThrow]
         .flatten
         .map(value => ResourceAttributes.ServiceName(value))
 
-      TelemetryResource(
+      val default = TelemetryResource.default
+      val fromEnv = TelemetryResource(
         Attributes.fromSpecific(attributes ++ serviceName.toSeq)
```

Comment:
I think you can get away without `.toSeq`?

```suggestion
        Attributes.fromSpecific(attributes ++ serviceName)
```

Reply:
Thanks, I've updated the code.